src/components/trace/hat-logbook-note.ts

```diff
@@ -1,14 +1,20 @@
-import { css, html, LitElement } from "lit";
+import { css, LitElement } from "lit";
 import { customElement, property } from "lit/decorators";
+import type { HomeAssistant } from "../../types";
 
 @customElement("hat-logbook-note")
 class HatLogbookNote extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
   @property() public domain = "automation";
 
   render() {
-    return html`
-      Not all shown logbook entries might be related to this ${this.domain}.
-    `;
+    return this.hass.localize(
+      "ui.panel.config.automation.trace.messages.not_all_entries_are_related_note",
+      {
+        domain: this.domain,
+      }
+    );
```

Comment:
What's the possible domains? Only script and automations, right? Should we add 2 translations for that because using a domain variable will not work in others language than english.

Reply:
automations and scripts indeed, according to the typing. 
I've added a fetch for the domain. Let me know if it is preferred to have 2 separate translations.

Reply:
If we only have 2 domains, I would prefer 2 translations. In french for example, we will have "**ce** script" and "**cette** automatisation".